### DISCUSSION TITLE
Anyway to get im2img to influence color/composition without controlnet processing that image?

### DISCUSSION THREAD

**Discussion by creeduk** [Mikubill/sd-webui-controlnet#689] [Q&A]
If the "Skip img2img processing when using img2img initial image" value is set then the image is ignored from what I can tell as changing it makes no difference so it works just like the txt2img only the controlnet values influence the prompt.

If that is unchecked then it seems to use controlnet 0 to try to process the image in the img2img window. Even if you say pick openpose and add a pose it will try to process the img2img. If I pick canny in controlnet0 and leave it blank and then something different on controlnet1 then it runs but often for me is out of memory so I was curious if there was some way that it would just let the image influence the noise as usual but not influence the controlnet, those settings only to be images directly added to their inputs?

**Comment by ljleb** [comment]
#683 will allow you to do what you want to do. Right now I don't think it is possible.

**Reply by creeduk** [reply]
That would be great.
